index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Awesome Books with ES6</title>
+  <link rel="stylesheet" href="./index.css">
+</head>
+
+<body>
+  <header id="header" class="d-flex row">
+    <div id="logo" class="d-flex row"><a>Awesome
+        Books</a>
+    </div>
+    <nav class="navbar">
+      <ul id="nav-list" class="d-flex row">
+        <li id="link-list" class="nav-item active"><a class="clickeable">List</a></li>
+        <li id="link-add" class="nav-item"><a class="clickeable">Add new</a></li>
+        <li id="link-contact" class="nav-item"><a class="clickeable">Contact</a></li>
+      </ul>
+    </nav>
+  </header>
+  <script type="module" src="./index.js"></script>
```

Comment:
* Kindly add the correct part to your javascript file, the index.js file is inside the `modules` folder so the correct path should be `/modules/index.js` currently because of this mistake i cant add nor remove books, but when i added the correct path, it worked.